lib/src/components/NxTransferList/NxTransferList.scss

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-container-helpers';
+@import '../../scss-shared/nx-text-helpers';
+
+// the padding on the sides of each transfer list control box
+$nx-transfer-list-control-box-side-padding: var(--nx-spacing-6x);
```

Comment:
It always bugs me when we set up variables named like this that we then use interchangeably as margin & padding. Why not `$nx-transfer-list-control-box-side-spacing`?

Reply:
> It always bugs me when we set up variables named like this that we then use interchangeably as margin & padding. Why not $nx-transfer-list-control-box-side-spacing?

18b08922